pkg/runctl/run_test.go

```diff
@@ -22,7 +22,9 @@ const (
 	completedFail             = `{"status": {"conditions": [{"message": "message1", "reason": "Failed", "status": "False", "type": "Succeeded"}], "steps": [{"name": "jenkinsfile-runner", "terminated": {"reason": "Error", "message": "ko", "exitCode": 1}}]}}`
 	completedValidationFailed = `{"status": {"conditions": [{"message": "message1", "reason": "TaskRunValidationFailed", "status": "False", "type": "Succeeded"}]}}`
 	//See issue https://github.com/SAP/stewardci-core/issues/? TODO: create public issue. internal: 21
-	timeout = `{"status": {"conditions": [{"message": "TaskRun \"steward-jenkinsfile-runner\" failed to finish within \"10m0s\"", "reason": "TaskRunTimeout", "status": "False", "type": "Succeeded"}]}}`
+	timeout              = `{"status": {"conditions": [{"message": "TaskRun \"steward-jenkinsfile-runner\" failed to finish within \"10m0s\"", "reason": "TaskRunTimeout", "status": "False", "type": "Succeeded"}]}}`
+	completionTimeSet    = `{"status": { "completionTime": "2019-05-14T08:24:49Z" } }`
+	completionTimeNotSet = `{"status": {}}`
```

Comment:
The avoid-using-vertical-space style makes the code hard to read. Please format it to be easily readable by humans.